cmd/deploy/deploy.go

```diff
@@ -111,6 +113,7 @@ type Command struct {
 	GetManifest          func(path string, fs afero.Fs) (*model.Manifest, error)
 	K8sClientProvider    okteto.K8sClientProviderWithLogger
 	Builder              builderInterface
+	Connector            buildCmd.BuildkitConnector
```

Comment:
Would it make sense to call it `RemoteConnector` or something similar. `Collector` is too generic and it is really hard to understand to what it refers being part of a deploy command object